modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::StarScallopRnaseq;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Analysis::Hive::Config::HiveBaseConfig_conf');
+
+sub default_options {
+  my ($self) = @_;
+  return {
+    # inherit other stuff from the base class
+    %{ $self->SUPER::default_options() },
+
+######################################################
+#
+# Variable settings- You change these!!!
+#
+######################################################
+########################
+# Misc setup info
+########################
+    'dbowner' => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name' => '' || $self->o('production_name') . '_' . $self->o('ensembl_release'),
+    'user_r'                    => '',                                                                                # read only db user
+    'user'                      => '',                                                                                # write db user
+    'password'                  => '',                                                                                # password for write db user
+    'server_set'                => '',                                                                                # What server set to user, e.g. set1
+    'pipe_db_server'            => '',                                                                                # host for pipe db
+    'databases_server'          => '',                                                                                # host for general output dbs
+    'dna_db_server'             => '',                                                                                # host for dna db
+    'pipe_db_port'              => '',                                                                                # port for pipeline host
+    'databases_port'            => '',                                                                                # port for general output db host
+    'dna_db_port'               => '',                                                                                # port for dna db host
+    'registry_host'             => '',                                                                                # host for registry db
+    'registry_port'             => '',                                                                                # port for registry db
+    'registry_db'               => '',                                                                                # name for registry db
```

Comment:
This is not needed, we don't use the registry

Reply:
Removed registry vars.